roles/cfg_openwrt/templates/common/config/freifunk.j2

```diff
@@ -2,8 +2,18 @@ config public 'contact'
 {% if contact_name is defined %}
 	option name '{{ contact_name }}'
 {% endif %}
+{% if nickname is defined %}
 	option nickname '{{ contact_nickname }}'
-	option mail '{{ contact_email }}'
+{% endif %}
+{% if contact_email is defined %}
+	{{ (contact_email_depr) | mandatory('The option "contact_email" in location/general.yml is deprecated. Please rename that flag to "contacts" and make sure to transform it into to a list.') }}
+{% elif (community is true) and (contacts is undefined)%}
+	option mail 'info+{{ inventory_hostname }}@berlin.freifunk.net'
+{% else %}
+{% for contact in (contacts) | mandatory('Please add at least one contact information via the contacts-flag in location/general.yml. Fellow Freifunkas need to be able to contact you in case they want to mesh with you!') %}
```

Comment:
Why do you put references to variables in () ?

Reply:
I think in that special case the reason is, that the mandatory directive should work on the `contacts`-variable only. I think I've seen that somewhere while googling. Not sure, if it will work the same without the brackets.